app/views/my_modules/repositories/_repositories_dropdown_list.html.erb

```diff
@@ -1,12 +1,16 @@
 <% @repositories.each do |repository| %>
   <% assigned_items_count = @my_module.repository_rows_count(repository) %>
-  <li class="repository">
+  <li class="repository"
+      data-table-url="<%= full_view_table_my_module_repository_path(@my_module, repository) %>"
+      data-assign-url-modal="<%= assign_repository_records_modal_my_module_repository_path(@my_module, repository) %>"
+      data-update-url-modal="<%= update_repository_records_modal_my_module_repository_path(@my_module, repository)%>" data-repository-id="<%= repository.id %>"
```

Comment:
What happened to words order here?

Reply:
Routes generate pattern - action name + controller name

Reply:
Where is this pattern defined?

Reply:
Inside rails, https://github.com/rails/rails/blob/master/guides/source/routing.md
It is default way how rails generate path helpers for routes. I simply use it.

Reply:
Did you read it?